boot.mount

```diff
@@ -1,6 +1,5 @@
 [Unit]
 ConditionKernelCommandLine=|ostree
-ConditionPathIsMountPoint=|/sysroot/boot
```

Comment:
Do we still need boot.mount at all? The right mount is set up by ostree in the initramfs, and if there's no boot.mount then systemd will hopefully not touch it beyond that point.